spec/requests/membership_submissions_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper"
+
+RSpec.describe "Membership submissions", type: :request do
+  before do
+    slack_link = "slack://channel?id=xxxx&team=yyyy"
+
+    allow_any_instance_of(ApplicationHelper).to receive(:link_to_open_slack)
```

Comment:
Instead of this, what do you think about creating the general slack channel? Feels like that might be better and would cover later cases where we may do something similar instead of overriding this one particular helper method which doesn't really have anything to do with membership submissions?

We could even go so far as to create that general slack channel in a before block in the rails helper so it's available to all tests.

Thoughts?

Reply:
Yeah, I agree creating the "general" slack channel is a much better approach.

> We could even go so far as to create that general slack channel in a before block in the rails helper so it's available to all tests.

Sure. Do we want it created for all tests? Won't it suffice to create it only for request specs?

Let me know if I'm missing something.